infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/send/LogReplicationAckReader.java

```diff
@@ -445,8 +445,9 @@ public void markSyncStatus(SyncStatus status) {
     /**
      * Start periodic replication status update task (completion percentage)
      */
-    public void startSyncStatusUpdatePeriodicTask() {
+    public void startSyncStatusUpdatePeriodicTask(SyncType toStateTransition) {
         log.info("Start sync status update periodic task");
+        lastSyncType = toStateTransition;
```

Comment:
the name `toStateTransition` is a bit confusing, I think just use `syncType` should be fine as it's already fetched before invoking this method...